### PR TITLE
add IndividualAttributeReport Response

### PR DESCRIPTION
**Pull request by tjikkun**

### CODE REVIEW

File: zigate/protocol/response.py [hunk 1]
```diff
@@ -144,6 +144,21 @@ def __str__(self):
 
 
 
+@register(0x8102)
+class IndividualAttributeReport(Response):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.struct = struct.Struct('!BHBHHBBH')
```
Comment (blubber) [inline zigate/protocol/response.py]: The struct isn't used elsewhere so it doesn't have to be an attribute of the object.

File: zigate/protocol/response.py [hunk 2]
```diff
@@ -144,6 +144,21 @@ def __str__(self):
...
+        self.seq_nr, self.src_addr, self.endpoint, self.cluster_id, self.attr_enum, self.attr_status, self.atrr_data_type, self.attr_size = self.struct.unpack(self.value[:self.struct.size])
```
Comment (blubber) [inline zigate/protocol/response.py]: This line is a bit long.

File: zigate/protocol/response.py [hunk 3]
```diff
@@ -144,6 +144,21 @@ def __str__(self):
...
+        logger.debug("%s %s" % (self.struct.size, len(self.value[self.struct.size:])))
```
Comment (blubber) [inline zigate/protocol/response.py]: The logger instance does it's own formatting which ignores errors: `logger.debug("%s %s", self.struct.size, len(self.value[self.struct.size:]))`
`

**Comment by tjikkun** [comment]
updated

**Pull request merged**